src/component/huskelapp/redigering/huskelapp-redigering.less

```diff
@@ -94,6 +94,6 @@
     }
     .arbeidsliste-innhold-tekst {
```

Comment:
Ser ut til at vi bruker samme klassen for både huskelappinnholdet og arbeidslisteinnholdet, jeg syns absolutt denne endringen ser bra ut på arbeidslisteinnholdet, men det gamle ser bedre ut på huskelappen, syns det blir for lite luft i bunnen vs på toppen. Hva med å lage en egen klasse til huskelappinnholdet? 😊